components/brave_today/browser/brave_news_controller.h

```diff
@@ -40,6 +40,8 @@ class HistoryService;
 
 namespace brave_news {
 
+constexpr char PADDED_IMAGE_URL_PATH_ENDING[] = ".pad";
```

Comment:
nit: If this is only used in brave_news_controller.cc`. I think it would be better to define at there in anon namespace.